tests/support/commands.ts

```diff
@@ -81,8 +96,12 @@ Cypress.Commands.add('createRuntimeConfig', ({ type } = {}): void => {
   }
 
   cy.findByLabelText(/object storage endpoint/i).type('http://0.0.0.0:9000');
+
+  if(type !== 'broken') {
```

Comment:
To address the current CI failure, we also need this condition on line 111 (don't reset the widget because the error dialog is on display)